server/views/pages/overview/partials/timelineTab/_timelineEvent-ACTION_PLAN_CREATED.njk

```diff
@@ -0,0 +1,18 @@
+<div class="moj-timeline__item" data-qa-event-type="ACTION_PLAN_CREATED">
```

Comment:
See the refactoring in `timelineTabContents.njk` below, but I've created a component and test for each event type that we render on the timeline.

Some of them are the same, some of them are subtly different, some of them are quite different. But it seemed sensible - especially when we're about to add 2 more - to have a separate component and test for each event type, even if there is a degree of duplication.

Reply:
I'm totally fine with duplication like this.